gvsbuild/projects.py

```diff
@@ -1841,3 +1847,133 @@ def build(self):
         self.builder.mod_env('GTK_BASE_DIR', self.builder.gtk_dir)
         self.builder.mod_env('PROMPT', '[ gvsbuild shell ] $P $G', subst=True)
         self.builder.exec_vs("cmd", working_dir=self.builder.working_dir)
+
+@project_add
+class Project_libsigplusplus(Tarball, Meson):
+    def __init__(self):
+        Project.__init__(self,
+            'libsig++',
+            archive_url = 'https://download.gnome.org/sources/libsigc++/2.10/libsigc++-2.10.0.tar.xz',
+            hash = 'f843d6346260bfcb4426259e314512b99e296e8ca241d771d21ac64f28298d81',
+            dependencies = [
+                'meson',
+                'ninja',
+            ]
+        )
+
+    def build(self):
+        Meson.build(self)

```

Comment:
does it have a license?

Reply:
Forgotten ;(

Updated with the license and, while I copy it, correct also the name, libsigc++